data/colors/vars/component_light.ts

```diff
@@ -210,6 +210,9 @@ export default {
   treeViewItem: {
     chevron: {
       hoverBg: alpha(get('scale.gray.2'), 0.32),
+    },
+    folder: {
+      fill: '#79B8FF'
```

Comment:
We currently use `scale-blue-3` for light mode directory icons in dotcom: https://cs.github.com/github/github/blob/e7f27d4cf8902dc3c9e6d8d908f066db54ab4020/app/assets/stylesheets/bundles/global/hx_color-modes.scss?q=org%3Agithub+--color-icon-directory#L34

```suggestion
      fill: get('scale.blue.3')
```